app/src/pages/instance/organizationsPage/organizationsPage.jsx

```diff
@@ -73,7 +74,7 @@ export const OrganizationsPage = () => {
       );
     }
 
-    return searchValue === null ? (
+    return searchValue === null && appliedFiltersCount === 0 ? (
```

Comment:
I suppose we need to think how to move this repeating logic to the `FilterButton` component.
Moreover, the search doesn't work after page refresh.

Reply:
I don't think it's possible to transfer this logic.